### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by dlukeomalley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +57,11 @@ $ pip3 install bento-cli
 
 Bento is for JavaScript, TypeScript, and Python projects. It requires Python 3.6+ and works on macOS Mojave (10.14) and Ubuntu 18.04+.
 
+## Motivations
+r2c is on a quest to make world-class security and bugfinding available to all developers, for free. We’ve learned that most developers have never heard of—let alone tried—tools that find deep flaws in code: like Codenomicon, which found Heartbleed, or Zoncolan at Facebook, which finds more top-severity security issues than any human effort. These tools find severe issues and also save tons of time, identifying hundreds of thousands of issues before humans can. Bento is a step towards universal access to tools like these. See [our Bento blog post](https://medium.com/@ievans/our-quest-to-make-world-class-security-and-bugfinding-available-to-all-developers-for-free-dce9eb7b06d0) to learn more.
```
Comment (ievans) [inline README.md]: we're missing all the links here

File: README.md [hunk 2]
```diff
@@ -63,7 +75,27 @@ To set aside preexisting results so you only see issues in new code:
 $ bento archive
 ```
 
-Bento is at its best when run automatically as a commit hook (i.e. `bento install-hook`) or as part of CI.
+Bento is at its best when run automatically as a git hook (i.e. `bento install-hook`) or as part of CI.
+
+### Command Line Options
```
Comment (ievans) [inline README.md]: this is good

File: README.md [hunk 3]
```diff
@@ -63,7 +75,27 @@ To set aside preexisting results so you only see issues in new code:
 $ bento archive
 ```
 
-Bento is at its best when run automatically as a commit hook (i.e. `bento install-hook`) or as part of CI.
+Bento is at its best when run automatically as a git hook (i.e. `bento install-hook`) or as part of CI.
```
Comment (ievans) [inline README.md]: specify "git pre-commit hook" IMO

File: README.md [hunk 4]
```diff
@@ -103,33 +135,19 @@ bento --agree --email <YOUR_EMAIL> check
 
 Please [open an issue](https://github.com/returntocorp/bento/issues/new?template=feature_request.md) if you need help setting up Bento with another CI provider. If you set up Bento with your provider of choice, we'd appreciate a PR to add instructions here!
```
Comment (pabloest) [inline README.md]:
```suggestion
Please [open an issue](https://github.com/returntocorp/bento/issues/new?template=feature_request.md) if you need help setting up Bento with another CI provider. If you set up Bento with your provider of choice, we’d appreciate a PR to add instructions here! 
```

File: README.md [hunk 5]
```diff
@@ -50,6 +57,13 @@ $ pip3 install bento-cli
...
+
+r2c is on a quest to make world-class security and bugfinding available to all developers, for free. We’ve learned that most developers have never heard of—let alone tried—tools that find deep flaws in code: like Codenomicon, which found [Heartbleed](http://heartbleed.com/), or Zoncolan at Facebook, which finds more [top-severity security issues](https://cacm.acm.org/magazines/2019/8/238344-scaling-static-analyses-at-facebook/fulltext) than any human effort. These tools find severe issues and also save tons of time, identifying [hundreds of thousands of issues](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/43322.pdf) before humans can. Bento is a step towards universal access to tools like these.
+
+We're also big proponents of opinionated tools like Black and Prettier. This has two implications: Bento ignores style-related issues and the bikeshedding that comes with them, and it ships with a curated set of checks that we believe are high signal and bug-worthy. See [Three things your linter shouldn’t tell you](https://blog.r2c.dev/posts/three-things-your-linter-shouldnt-tell-you/) for more details on our decision making process.
```
Comment (pabloest) [inline README.md]:
```suggestion
We’re also big proponents of opinionated tools like Black and Prettier. This has two implications: Bento ignores style-related issues and the bikeshedding that comes with them, and it ships with a curated set of checks that we believe are high signal and bug-worthy. See [Three things your linter shouldn’t tell you](https://blog.r2c.dev/posts/three-things-your-linter-shouldnt-tell-you/) for more details on our decision making process.
```

**Pull request merged**